.azure_pipelines/ci-pipeline-makefile.yml

```diff
@@ -75,6 +75,8 @@ jobs:
           DB_NAME: $(DB_NAME_USEAST)
           DB_SERVER_NAME: $(DB_SERVER_NAME_USEAST)
           MAA_URL: $(DB_MAA_URL_USEAST)
+          # If set, don't run certain tests
+          MYST_SKIP_PR_TEST: 1
```

Comment:
Is this configuration only applied in PR pipeline? The name of the configuration is confusing.